sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -311,7 +311,9 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
       EGL10.EGL_SUCCESS -> { }
       EGL11.EGL_CONTEXT_LOST -> {
         logW(TAG, "Context lost. Waiting for re-acquire")
-        releaseEgl()
+        // Android surface is still valid so we don't release it
+        // otherwise no more rendering will happen
```

Comment:
Is this actually possible, EGL_CONTEXT_LOST and valid surface? Isn't it just happening due to the way you conduct the test?

Reply:
fixed things a bit, PTAL